prometheus_es_exporter/__init__.py

```diff
@@ -304,6 +305,8 @@ def main():
                         help='disable query monitoring. Config file does not need to be present if query monitoring is disabled.')
     parser.add_argument('-c', '--config-file', default='exporter.cfg',
                         help='path to query config file. Can be absolute, or relative to the current working directory. (default: exporter.cfg)')
+    parser.add_argument('--config-dir', default='./config',
+                        help='path to query config directory. All config files in the directory will be included. Can be absolute, or relative to the current working directory. (default: ./config)')
```

Comment:
Would be good to include something in the help text about the order files are merged in, i.e. first `--config-file`, then `--config-dir` files, in alphabetical order.